docs/middleware/neutrino-middleware-minify/README.md

```diff
@@ -58,15 +58,15 @@ module.exports = {
 module.exports = {
   use: [
     ['neutrino-middleware-minify', {
-      babili: {},
-      overrides: {}
+      minify: {},
+      plugin: {}
     }]
   ]
 };
 ```
 
-The `babili` and `overrides` properties map to the options defined by
-[babili-webpack-plugin](https://www.npmjs.com/package/babili-webpack-plugin#options).
+The `minify` and `plugin` properties map to the options defined by
+[babel-minify-webpack-plugin](https://www.npmjs.com/package/babel-minify-webpack-plugin#options).
```

Comment:
The `#options` anchor tag doesn't appear to work on the NPM package page. Perhaps we should link to https://github.com/webpack-contrib/babel-minify-webpack-plugin#options instead?

Reply:
I'm cool with doing that.

Reply:
Fixed in #340